src/routes/form/+page.svelte

```diff
@@ -23,4 +25,61 @@
 		<option value="9">Sem IX</option>
 		<option value="10">Sem X</option>
 	</select>
+	<label for="role">Roll No</label>
+	<input type = "text" id = "role" name = "role">
+	<label for="category">Category</label>
+	<select id="category">
+		<option value="1">General</option>
+		<option value="2">EWS</option>
+		<option value="3">OBC</option>
+		<option value="4">WDP</option>
+		<option value="5">SC</option>
+		<option value="6">ST</option>
+		<option value="7">VHC</option>
+		<option value="8">PHC</option>
+	</select>
+	<label for="phone">Phone No</label>
+	<input type = "tel" id = "phone" name = "phone">
+
+	
+	<strong>Details of the fees payment</strong>
+	<br>
+	<label for="acc-holder">Name of the account holder</label>
+	<input type="text" id="acc-holder" name="acc-holder">
+	<label for="acc-no">Account No</label>
+	<input type="number" id="acc-no" name="acc-no">
+	<label for="bank">Name of the bank</label>
+	<input type="text" id="bank" name="bank">
+	<label for="branch">Branch</label>
+	<input type="text" id="branch" name="branch">
+
+	<label for="ifsc">IFSC code</label>
+	<input type="number" id="ifsc" name="ifsc">
+	<label for="paid-amt">Amount paid</label>
+	<input type="number" id="paid-amt" name="paid-amt">
+	<label for="pay-date">Date of Payment</label>
+	<input type="date" id="pay-date" name="bank">
+	<label for="mode">Payment Mode</label>
+	<select id="mode">
+		<option value="1">NEFT</option>
+		<option value="2">SBI to SBI</option>
+		<option value="3">Debit card</option>
+		<option value="4">Credit card</option>
+	</select>
+	<label for="transaction-no">Transaction Number</label>
+	<input type="number" id="transaction-no" name="transaction-no">
+	<label for="reference-no">Reference Number</label>
+	<input type="number" id="reference-no" name="reference-no">
+	<label for="utr">UTR Number</label>
+	<input type="number" id="utr" name="utr">
+	<label for="inb">INB Reference Number (for transfer from SBI to SBI)</label>
+	<input type="number" id="inb" name="inb">
+
+	<strong>Details of the fees payment</strong>
+	<br>
+
+	<ol type = "a">
```

Comment:
type a?

Reply:
From what I recollect, type = "a" orders the list indexes as a. b. c. and so on

Reply:
hahaha. no. that's a CSS prop. `style="list-style-type: lower-roman-latin`. or setting it in CSS. in tailwind it's `class="...list-[lower-latin]...`

but eitherway, not required. just remove it